RPGManager_development/test/unit_test/character_sheet_unit_tests/ArmorsUnitTests.java

```diff
@@ -0,0 +1,145 @@
+		int actualArmorValue = characterSheet.getData(Fields.ARMOR_RATING);
+		assertEquals(4, actualArmorValue + characterSheet.<Armors>getData(Fields.ARMOR_TYPE).getArmorRating());
+	}
+
+	@Test
+	public void testGetDefaultArmorRatingWarrior() {
+		CharacterSheet characterSheet = new CharacterSheet("characterSheet");
+		characterSheet.setData(Fields.BASECLASS, BaseClasses.WARRIOR);
+		int actualArmorValue = characterSheet.getData(Fields.ARMOR_RATING);
+		assertEquals(4, actualArmorValue + characterSheet.<Armors>getData(Fields.ARMOR_TYPE).getArmorRating());
+	}
+
+	@Test
+	public void testGetDefaultArmorRatingRogue() {
+		CharacterSheet characterSheet = new CharacterSheet("characterSheet");
+		characterSheet.setData(Fields.BASECLASS, BaseClasses.ROGUE);
+		int actualArmorValue = characterSheet.getData(Fields.ARMOR_RATING);
+		assertEquals(3, actualArmorValue + characterSheet.<Armors>getData(Fields.ARMOR_TYPE).getArmorRating());
+	}
+
+	@Test
+	public void testGetDefaultArmorRatingMage() {
+		CharacterSheet characterSheet = new CharacterSheet("characterSheet");
+		characterSheet.setData(Fields.BASECLASS, BaseClasses.MAGE);
+		int actualArmorValue = characterSheet.getData(Fields.ARMOR_RATING);
+		assertEquals(0, actualArmorValue + characterSheet.<Armors>getData(Fields.ARMOR_TYPE).getArmorRating());
+	}
+
+	// Functional tests
+
+	@Test
+	public void testSetArmorTypeLightPlate() {
+		CharacterSheet characterSheet = new CharacterSheet("characterSheet");
+		Armors expectedArmor = Armors.LIGHT_PLATE;
+		characterSheet.setData(Fields.ARMOR_TYPE, expectedArmor);
+		Armors actualValue = characterSheet.getData(Fields.ARMOR_TYPE);
+		assertEquals(Armors.LIGHT_PLATE, actualValue);
+	}
+
+	@Test
+	public void testSetArmorForWarrior() {
+		CharacterSheet characterSheet = new CharacterSheet("characterSheet");
+		characterSheet.setData(Fields.BASECLASS, BaseClasses.WARRIOR);
+		Armors expectedArmor = Armors.HEAVY_MAIL;
+		characterSheet.setData(Fields.ARMOR_TYPE, expectedArmor);
+		Armors actualValue = characterSheet.getData(Fields.ARMOR_TYPE);
+		assertEquals(Armors.HEAVY_MAIL, actualValue);
+	}
+
+	@Test
+	public void testSetArmorForRugue() {
+		CharacterSheet characterSheet = new CharacterSheet("characterSheet");
+		characterSheet.setData(Fields.BASECLASS, BaseClasses.WARRIOR);
+		Armors expectedArmor = Armors.HEAVY_MAIL;
+		characterSheet.setData(Fields.ARMOR_TYPE, expectedArmor);
+		Armors actualValue = characterSheet.getData(Fields.ARMOR_TYPE);
+		assertEquals(Armors.HEAVY_MAIL, actualValue);
+	}
+
+	@Test
+	public void testSetArmorForMage() {
+		CharacterSheet characterSheet = new CharacterSheet("characterSheet");
+		characterSheet.setData(Fields.BASECLASS, BaseClasses.MAGE);
+		Armors expectedArmor = Armors.HEAVY_MAIL;
+		characterSheet.setData(Fields.ARMOR_TYPE, expectedArmor);
+		Armors actualValue = characterSheet.getData(Fields.ARMOR_TYPE);
+		assertEquals(Armors.HEAVY_MAIL, actualValue);
+	}
+
+	// Test attributes with penaltys for warriror
+
+	@Test

```

Comment:
Ennek tulajdonképpen nincs köze a DEXTERITY-hez hiszen az nem változik. Az armor penalty értékét teszteled vele, de azt már teszteled a parametrizált tesztekben is. Itt mit szerettél volna csinálni? Magyarázd el, hátha tudok segíteni.